test/e2e/specs/all-features/record/presentationRelatedTab.spec.js

```diff
@@ -0,0 +1,91 @@
+        { title: "Description", type: "markdown", value: "<p><strong>CARING. SHARING. DARING.</strong><br>\nRadisson<sup>®</sup> is synonymous with outstanding levels of service and comfort delivered with utmost style. And today, we deliver even more to make sure we maintain our position at the forefront of the hospitality industry now and in the future.<br>\nOur hotels are service driven, responsible, socially and locally connected and demonstrate a modern friendly attitude in everything we do. Our aim is to deliver our outstanding <code>Yes I Can!</code> <sup>SM</sup> service, comfort and style where you need us.</p>\n<p><strong>THE RADISSON<sup>®</sup> WAY</strong> Always positive, always smiling and always professional, Radisson people set Radisson apart. Every member of the team has a dedication to <code>Yes I Can!</code> <sup>SM</sup> hospitality – a passion for ensuring the total wellbeing and satisfaction of each individual guest. Imaginative, understanding and truly empathetic to the needs of the modern traveler, they are people on a special mission to deliver exceptional Extra Thoughtful Care.</p>\n"},
+        { title: "Number of Rooms", value: "23", type: "int2"},
+        { title: "Cover Image", value: "3,005", type: "int2", presentation: { type: "url", template: "{{{chaise_url}}}/record/#{{catalog_id}}/product-record:file/id=3005"} },
+        { title: "Thumbnail", value: null, type: "int4"},
+        { title: "Operational Since", value: "2008-12-09 00:00:00", type: "timestamptz" },
+        { title: "Is Luxurious", value: "true", type: "boolean" },
+        { title: "json_col", value:'<pre>'+JSON.stringify(null,undefined,2)+'</pre>'}
+    ],
+    no_related_data: {
+        key: {
+            name: "id",
+            value: "4004",
+            operator: "="
+        },
+        tables_order: ["booking (no results found)", "accommodation_image (no results found)", "media (no results found)"]
+    }
+};
+
+describe('Collapse Related tables, ', function() {
+
+        beforeAll(function() {
+            var keys = [];
+            browser.ignoreSynchronization = true;
+            keys.push(testParams.key.name + testParams.key.operator + testParams.key.value);
+
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/product-record:" + testParams.table_name + "/" + keys.join("&");
+            browser.get(url);
+            accordionSet = element.all(by.css('.related-table-heading'));
+            chaisePage.waitForElement(chaisePage.recordPage.getEntityTitleElement(), browser.params.defaultTimeout);
+        });
+
+        it('should collapse related tables after cutoff value',function(){
+            browser.executeScript("return chaiseConfig;").then(function(chaiseConfig) {
+            accordionSet.count().then(function(accrCount){
```

Comment:
You should use promise chaining here instead of nested promises.
```
browser.executeScript().then(function() {
  return accordionSet.count();
}).then(function() {
  expectation
});
```
And then you should attach a `.catch()` to the end of that promise chain in case something errors out.